### PR TITLE
Update az.po

### PR DESCRIPTION
**Pull request by manymous**

### CODE REVIEW

File: po/az.po [hunk 1]
```diff
@@ -7,56 +7,60 @@ msgstr ""
...
 msgid "Close"
-msgstr ""
+msgstr "Bağla"
 
 #: ../src/PreferencesWindow.vala:51
 msgid ""
 "<b>Note:</b> From now on, Clipped will automatically start in the background "
 "when you log in."
 msgstr ""
+"<b> Qeyd: </ b> İndi, Clipped avtomatik olaraq arxa fonda başlayacaq"
```
Comment (fitojb) [inline po/az.po:45]: Please remove the stray spaces in the XML tag, like this:

```suggestion
"<b>Qeyd:</b> İndi, Clipped avtomatik olaraq arxa fonda başlayacaq"
```

File: po/az.po [hunk 2]
```diff
@@ -7,56 +7,60 @@ msgstr ""
...
+"daxil olduğunuzda."
 
 #: ../src/PreferencesWindow.vala:52
 msgid ""
 "If you wish to change this behaviour, visit <a href=\"settings://"
 "applications/startup\">Application Settings…</a>"
 msgstr ""
+"Bu davranışı dəyişdirmək istəyirsinizsə, <a href=\"settings://"
+"applications/startup\">Application Settings…</a> ziyarət edin"
```
Comment (fitojb) [inline po/az.po:54]: Please translate “Application Settings…” inside the <a> link

File: po/az.po [hunk 3]
```diff
@@ -7,56 +7,60 @@ msgstr ""
...
 
 #: ../src/PreferencesWindow.vala:66
 msgid "General Settings"
-msgstr ""
+msgstr "Ümumi Parametrlər"
 
 #: ../src/PreferencesWindow.vala:79
 msgid "Paste Shortcut:"
-msgstr ""
+msgstr "Kısayolları yerləşdirin"
```
Comment (fitojb) [inline po/az.po:62]: Missing colon at the end of the string:
```suggestion
msgstr "Kısayolları yerləşdirin:"
```

**Review by fitojb — CHANGES_REQUESTED** [review]

**Comment by davidmhewitt** [comment]
I'm going to close this pull request as the improvements suggested in the review (thanks @fitojb) have not been made, and I don't have write access to this branch to make them myself.

If someone is still available to improve this translation, please open another pull request.

**Pull request closed without merging**